pom.xml

```diff
@@ -41,27 +41,133 @@
 
     <name>Apache Sling Scripting HTL Compiler</name>
 
-    <description>
-        The Apache Sling Scripting HTL Compiler provides support for compiling HTML Template Language scripts into an Abstract
-        Syntax Tree.
-    </description>
+    <description>The Apache Sling Scripting HTL Compiler provides support for compiling HTML Template Language scripts into an Abstract
+        Syntax Tree.</description>
 
     <scm>
         <connection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-scripting-sightly-compiler.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-scripting-sightly-compiler.git</developerConnection>
-        <url>https://github.com/apache/sling-org-apache-sling-scripting-sightly-compiler.git</url>
         <tag>org.apache.sling.scripting.sightly.compiler-1.2.6-1.4.0</tag>
+        <url>https://github.com/apache/sling-org-apache-sling-scripting-sightly-compiler.git</url>
     </scm>
 
     <properties>
```

Comment:
I would suggest setting an explicit java bytecode version (no need to upgrade > 8 for now)